.release/ci.hcl

```diff
@@ -18,159 +18,23 @@ project "nomad" {
   }
 }
 
-event "merge" {
-  // "entrypoint" to use if build is not run automatically  // i.e. send "merge" complete signal to orchestrator to trigger build
-}
-
-event "build" {
-  depends = ["merge"]
-
-  action "build" {
-    organization = "hashicorp"
-    repository   = "nomad"
-    workflow     = "build"
-  }
-}
-
```

Comment:
Removing the `build` event?  The new `prepare` event explicitly depends on `build`.

Reply:
This is why code reviews are important. 😆  
I was/am still planning to test this branch after the prepare workflow is green again. But thanks for catching this ahead of time!!